Misc/Lobby/oa_tanabata.csv

```diff
@@ -133,18 +133,18 @@ result56#1,"""クリきんと"""
 result57#1,"""mashiro@uyo"""
 result58#1,"""Cool _Nicos"""
 result59#1,"""レイ・ブランフォード"""
-result60#1,"""thebluecat"""
+result60#1,"""Thebluecat"""
 result61#1,"""間取"""
-result62#1,"""ashitubo3"""
+result62#1,"""Ashitubo3"""
 result63#1,"""雛月ユウキ"""
 result64#1,"""kise-yayoi"""
 result65#1,"""Qreuz"""
-result66#1,"""lakota"""
+result66#1,"""Lakota"""
```

Comment:
Still unfixed ( just discard modifications on the file)

Reply:
done